docs/usage/login.md

```diff
@@ -0,0 +1,32 @@
+Before you can login you will need to:
+
+- [install CloudFoundry command line tools](/usage/setup/)
+- be a member of an existing project on cloud.gov.au
+
+If you are starting a new project then you will need to [Request a new project](/usage/start_new_project/)
+
+If you need access to an existing project then ask the projects CloudFoundry maintainer. This is usually the Delivery Manager or Tech Lead. If you get stuck you can [send us a support ticket](mailto:support@cloud.gov.au).
+
+Login with:
+
+```
+cf login -a https://api.system.staging.digital.gov.au --skip-ssl-validation
```

Comment:
Also, testing it, the login command prompts for the org and space so you don't need to run `cf target`. You only need to run `cf target` to change org and space. You can also pass the username/org/space as commandline arguments to `cf login` but maybe that's overdoing it on complexity. My normal login command is 
```
cf login -a https://api.system.staging.digital.gov.au -u <email> -o <org> -s <space>
```
just because I get annoyed responding to the prompts